screwdriver.yaml

```diff
@@ -14,14 +14,14 @@ jobs:
             - vet: go vet ./...
             - gofmt: (! gofmt -d . | grep '^')
             # pin gotestsum version because v1.12.3 occurs error when install
-            - test-setup: go install gotest.tools/gotestsum@v1.12.2
+            - test-setup: go install gotest.tools/gotestsum@v1.13.0
```

Comment:
small comment. If the bug in gotestsum has been fixed, it might be okay to use the latest tag again as before.
https://github.com/screwdriver-cd/launcher/commit/23dbd06b01fe059223abc7c2e5dee357a6af149a